bootstrap/postgres/init.go

```diff
@@ -83,6 +83,14 @@ func Migration() *migrate.MemoryMigrationSource {
 					`ALTER TABLE IF EXISTS channels RENAME COLUMN mainflux_channel TO magistrala_channel`,
 				},
 			},
+			{
+				Id: "configs_5",
+				Up: []string{
+					`ALTER TABLE IF EXISTS configs RENAME COLUMN owner TO domain_id`,
+					`ALTER TABLE IF EXISTS channels RENAME COLUMN owner TO domain_id`,
+					`ALTER TABLE IF EXISTS configs ADD CONSTRAINT configs_name_domain_id_key UNIQUE (name, domain_id)`,
```

Comment:
There is no need for these two `channel_domain_id` and `config_domain_id`, because both chanenl and thing should be in same domain. 
We should impose this validation, while bootstrap creation.
We will get the channel and thing domain id while retrieving their information form things service. 
There we can validate both thing and channel are belong to same domain.

Here please add new column `domain_id` and remove  `channel_domain_id` , `config_domain_id`